lib/matplotlib/cbook/deprecation.py

```diff
@@ -24,16 +24,16 @@ class MatplotlibDeprecationWarning(UserWarning):
 def _generate_deprecation_warning(
         since, message='', name='', alternative='', pending=False,
         obj_type='attribute', addendum='', *, removal=''):
-
```

Comment:
This patch doesn't change the logic, just clarifies that pending deprecations have no scheduled removal.